tests/components/QuestionsAndAnswerTests/QAList.test.js

```diff
@@ -45,13 +47,39 @@ afterAll(()=> server.close())
 
 describe('QA List', () => {
   test('Should render the Q/A List', () => {
+    var app;
+    act(()=>{
```

Comment:
I think we determined if you are receiving an ACT warning, it is because the component is preforming some async action so we should be using the findBy query methods instead of queryBy and getBy.

Reply:
Second this!